### PR TITLE
docs: TrustBloc sandbox playground demo instructions

### PR DESCRIPTION
**Pull request by rolsonquadras**
Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: docs/demo/sandbox_playground.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+## Components
+1. [Wallet](https://agent.sandbox.trustbloc.dev/dashboard)
+2. [Issuer](https://demo-issuer.sandbox.trustbloc.dev/)
+3. [Verifier](https://demo-verifier.sandbox.trustbloc.dev/) 
+
+## Steps
+1. Register a Wallet identity : Go to [DID Management](https://agent.sandbox.trustbloc.dev/DIDManagement) and 
+create a new TrustBloc DID by selecting the key type (Ed25519 or P256) and signature type (Ed25519Signature2018 or 
+JwsSignature2020). Type in a friendly name and click save. On successful, save the new DID shows up in the table. Refer [this](#wallet-dids)
```
Comment (sandrask) [inline docs/demo/sandbox_playground.md:13]: If successful or remove "On successful"

Comment (rolsonquadras) [reply docs/demo/sandbox_playground.md:13]: thanks. will re-align the sentence.

Comment (rolsonquadras) [reply docs/demo/sandbox_playground.md:13]: looks like i misplaced the comma.

File: docs/demo/sandbox_playground.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+2. [Issuer](https://demo-issuer.sandbox.trustbloc.dev/)
+3. [Verifier](https://demo-verifier.sandbox.trustbloc.dev/) 
+
+## Steps
+1. Register a Wallet identity : Go to [DID Management](https://agent.sandbox.trustbloc.dev/DIDManagement) and 
+create a new TrustBloc DID by selecting the key type (Ed25519 or P256) and signature type (Ed25519Signature2018 or 
+JwsSignature2020). Type in a friendly name and click save. On successful, save the new DID shows up in the table. Refer [this](#wallet-dids) 
+for more info on supported DIDs.
+2. Register the Wallet (CHAPI) : Go to [Register Wallet](https://agent.sandbox.trustbloc.dev/RegisterWallet) and click 
+on "Allow" when a pop-up asks for the permission.
```
Comment (sandrask) [inline docs/demo/sandbox_playground.md:16]: different format used for allow button comparing to text bellow. Also button actions should all start with either capital or lowercase (just to be consistent)

File: docs/demo/sandbox_playground.md (hunk 1 above)
Comment (rolsonquadras) [reply docs/demo/sandbox_playground.md]: updated.

File: docs/demo/sandbox_playground.md (hunk 2 above)
Comment (rolsonquadras) [reply docs/demo/sandbox_playground.md:16]: You mean the button texts on the actual Demo UI or this document ?

Comment (sandrask) [reply docs/demo/sandbox_playground.md:16]: I didn't notice it in demo, I am just commenting for this text for example "Allow" vs. `Allow` and 'submit' and 'cancel' If this is how it is in demo I am ok with it. It is just noticable in text that they are different formats.

Comment (rolsonquadras) [reply docs/demo/sandbox_playground.md]: [Updated](https://github.com/trustbloc/edge-sandbox/compare/64fc99f1539a02b1e669c96b85ffb71e80f4f9fc..2ecd134f9a1d420ea2501caad98c188149e58f26#diff-49bb0845ac16bfdf9fd7834b51d22e19R16) the formatting.

Agree on consistency of UI texts. may be we can revisit egde-agent and make sure we are consistent.

**Pull request merged**